source/_integrations/drop_connect.markdown

```diff
@@ -0,0 +1,60 @@
+---
+title: DROP
+description: Instructions on how to integrate DROP into Home Assistant.
+ha_category:
+  - Sensor
+ha_config_flow: true
+ha_codeowners:
+  - '@ChandlerSystems'
+  - '@pfrazer'
+ha_domain: drop_connect
+ha_iot_class: Local Push
+ha_mqtt: true
+ha_release: '2024.1'
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The DROP integration provides connectivity with
+[DROP smart water system](https://dropconnect.com) devices via a local MQTT API.
+
+### Supported devices
+
+There is currently support for the following DROP products within Home Assistant:
+
+- **DROP Hub**: centralized system status and control.
+- **Leak Detector**: reports detected leaks around the home.
+- **Softener**: improves water quality by decreasing water hardness with integral water shutoff.
+- **Filter**: improves water quality by removing contaminants with integral water shutoff.
+- **Protection Valve**: automatic water shutoff with slow leak detection.
+- **Pump Controller**: smart replacement for an FSG pressure switch.
+- **RO Filter**: reverse osmosis drinking water filtration.
+- **Salt Sensor**: alerts when the salt level in the softener brine tank is low.
+
+### Configuration
```

Comment:
This is not integration configuration, but device configuration.

This section should be called "Pre-requisites" instead.

Please address this in a new PR

Reply:
Address in #30527